e2etest/zt_newe2e_blobfs_test.go

```diff
@@ -0,0 +1,244 @@
+package e2etest
+
+import (
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"strconv"
+)
+
+func init() {
+	suiteManager.RegisterSuite(&BlobFSTestSuite{})
+}
+
+type BlobFSTestSuite struct{}
+
+func (s *BlobFSTestSuite) Scenario_UploadFile(svm *ScenarioVariationManager) {
+	fileName := "test_1k.txt"
+	body := NewRandomObjectContentContainer(svm, common.KiloByte)
+
+	srcObj := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Local()), ResourceDefinitionContainer{}).
+		GetObject(svm, fileName, common.EEntityType.File())
+	srcObj.Create(svm, body, ObjectProperties{})
+
+	acct := GetAccount(svm, PrimaryHNSAcct)
+	dstService := acct.GetService(svm, common.ELocation.BlobFS())
+	dstContainer := CreateResource[ContainerResourceManager](svm, dstService, ResourceDefinitionContainer{})
+
+	RunAzCopy(svm, AzCopyCommand{
+		Verb:    AzCopyVerbCopy,
+		Targets: []ResourceManager{srcObj, dstContainer.(RemoteResourceManager).WithSpecificAuthType(ResolveVariation(svm, []ExplicitCredentialTypes{EExplicitCredentialType.SASToken(), EExplicitCredentialType.OAuth()}), svm, CreateAzCopyTargetOptions{})},
+		Flags: CopyFlags{
+			CopySyncCommonFlags: CopySyncCommonFlags{
+				Recursive: pointerTo(true),
+			},
+		},
+	})
+
+	ValidateResource[ObjectResourceManager](svm, dstContainer.GetObject(svm, fileName, common.EEntityType.File()), ResourceDefinitionObject{
+		Body: body,
+	}, true)
+}
+
+func (s *BlobFSTestSuite) Scenario_UploadFile64MB(svm *ScenarioVariationManager) {
```

Comment:
File size could be a variation, another valid question is: why are we doing a specific test of 64MB?